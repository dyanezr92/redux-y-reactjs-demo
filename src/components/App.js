import React from 'react'
import Col from './Col'
import Header from '../containers/Header'
import Comments from '../containers/Comments'
import Complaints from '../containers/Complaints'
import finn from '../static/finn.gif'

const App = () => (
  <Col w={12}>
    <Header title="Awesome Finn" src={ finn } />
    <Comments />
    <Complaints />
  </Col>
)

export default App
