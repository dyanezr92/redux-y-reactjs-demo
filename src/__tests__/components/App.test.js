import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store'; // Smart components

import App from '../../components/App';

describe('<App/>', () => {
  describe('render()', () => {
    test('renders the App', () => {
      const wrapper = shallow(<App/>);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
