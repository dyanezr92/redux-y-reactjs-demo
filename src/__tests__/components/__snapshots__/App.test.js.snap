// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> render() renders the App 1`] = `
<Col
  w={12}
>
  <Connect(HeaderWithPicture)
    src="finn.gif"
    title="Awesome Finn"
  />
  <Connect(UserTextSection) />
  <Connect(UserTextSection) />
</Col>
`;
